.npmrc

```diff
@@ -4,3 +4,4 @@
 # This file sets the default parameters for manual `pnpm install`.
 #
```

Comment:
You should not delete package-lock.json. That is also needed...

Reply:
There is still no package-lock

Reply:
really?, I see that file in the diff, or am I missing something

![Screenshot 2022-07-22 at 16 53 55](https://user-images.githubusercontent.com/161853/180466019-b430982c-78d8-43ba-b35c-08398f099408.png)